README.md

```diff
@@ -12,3 +12,5 @@ how to use:
 
 my setup is ec2 in aws.
 I didn't use Docker, so if you want your project to run on linux (at least ubuntu) you need to install all the packages in the Docker file.
+
+Hi I'm Daniel and I created this branch

```

Comment:
WE DON'T CARE ABOUT YOU.
REMOVE THIS SHIT

Reply:
My therapist told me it's healthy tho